### DISCUSSION TITLE
Date Range Picker

### DISCUSSION THREAD

**Discussion by siddharth1903** [vmware/clarity#6088] [Ideas] [answered]
Is there a plan to add a date range picker component? It would really help the consumers of this library to stick to a single design system for standard components! Especially for theming, maintability and accessibility!

**Comment by mathisscott** [comment, marked as answer]
Hi @siddharth1903 .

As we move more towards the web component architecture, we are not prioritizing new features for the pure Angular components. This means that a date range picker will not be introduced for the legacy Angular datepicker component. 

When we build out a datepicker for Clarity Core, however, features such as these are still an open discussion. There is value to keeping it fresh in our minds as we move forward. Thanks!

**Reply by d-m-s** [reply]
some great ideas posted some while ago:
https://github.com/vmware/clarity/issues/474 Date & Time Picker component (new)

Any news on this one? Any Roadmap?

**Reply by d-m-s** [reply]
**Additional inspiration regarding range-controls from Material Design:**
<img width="175" alt="2022-03-02_17h10_32" src="https://user-images.githubusercontent.com/311619/156401395-55ba8100-1aeb-41e4-b9bd-0defecde9048.png">

<img width="231" alt="2022-03-02_17h10_43" src="https://user-images.githubusercontent.com/311619/156401397-54325d59-90b5-4af4-a7a8-bc01d6691070.png">

<img width="119" alt="2022-03-02_17h11_04" src="https://user-images.githubusercontent.com/311619/156401399-b731e2f7-9721-4160-bd34-dbe4955e8b7c.png">

**Comment by mathisscott** [comment]
Hi @d-m-s 

Any questions about our roadmap should be directed towards our new manager. The following is a [repost from a previous answer](https://github.com/vmware/clarity/discussions/6710#discussioncomment-2204847).

> Our new manager, Steve, sets the priorities for the team so I recommend reaching out to him at [strefethen@vmware.com](mailto:strefethen@vmware.com) with any questions about priorities or release timelines. If you work on a VMware product, I would also recommend including that information as well as your chain of command to raise up any concerns.
